roles/cis/tasks/3/3.1.1.yml

```diff
@@ -13,7 +13,7 @@
             touch {{ security_hardening_conf }}
         fi
       delegate_to: localhost
```

Comment:
can we remove this?

Reply:
Pre-existing /etc/sysctl.d/50-security-hardening.conf is outside of chroot, i.e., this file is in photon vm. If it is not delegated to localhost, it complains that file is not accessible/not present.
If written to /etc/sysctl.conf did not see any issues.

Reply:
Fixed!